### PR TITLE
Clarify SignedMessage documentation [ECR-3478]

### PR DESCRIPTION
**Pull request by aleksuss**
---

See: https://jira.bf.local/browse/ECR-3478

### Definition of Done

- [ ] There are no TODOs left in the merged code
- [ ] Change is covered by automated tests
- [ ] Benchmark results are attached (if applicable)
- [ ] The [coding guidelines] are followed
- [ ] Public API has proper documentation
- [ ] Changelog is updated if needed (in case of notable or breaking changes)
- [ ] The continuous integration build passes

[coding guidelines]: https://github.com/exonum/exonum/blob/master/CONTRIBUTING.md#conventions

### CODE REVIEW

File: exonum/src/proto/schema/exonum/consensus.proto [hunk 1]
```diff
@@ -29,11 +29,11 @@ import "google/protobuf/timestamp.proto";
...
 // hash(SignedMessage) = hash(payload || key.bytes || sign.bytes)
 message SignedMessage {
-  // Message payload.
+  // Payload of the message.
   bytes payload = 1;
-  // Message author.
+  // Public key of the author of this message.
   exonum.crypto.PublicKey author = 2;
-  // Digital signature.
+  // Digital signature of the payload which creates with a secret key of the author of this message.
```
Comment (popzxc) [inline exonum/src/proto/schema/exonum/consensus.proto]:
```suggestion
  // Digital signature of the `payload` created by a `SecretKey` of the author of the message.
```

File: exonum/src/messages/types.rs [hunk 2]
```diff
@@ -35,11 +35,11 @@ use exonum_proto::ProtobufConvert;
...
 #[protobuf_convert(source = "consensus::SignedMessage")]
 pub struct SignedMessage {
-    /// Message payload.
+    /// Payload of the message.
     pub payload: Vec<u8>,
-    /// Message author.
+    /// `PublicKey` of the author of this message.
     pub author: PublicKey,
-    /// Digital signature.
+    /// Digital signature of the `payload` which creates with a `SecretKey` of the author of this message.
```
[the remark this answers is not in the document]
Comment (Maria-Nosyk) [reply exonum/src/messages/types.rs]:
```suggestion
    /// Digital signature over `payload` created with `SecretKey` of the author of the message.
```

File: exonum/src/proto/schema/exonum/consensus.proto [hunk 3]
```diff
@@ -29,11 +29,11 @@ import "google/protobuf/timestamp.proto";
 // Keep in mind, that:
 // hash(SignedMessage) = hash(payload || key.bytes || sign.bytes)
 message SignedMessage {
-  // Message payload.
+  // Payload of the message.
   bytes payload = 1;
-  // Message author.
+  // Public key of the author of this message.
```
Comment (Maria-Nosyk) [inline exonum/src/proto/schema/exonum/consensus.proto]:
```suggestion
  // Public key of the author of the message.
```

File: exonum/src/proto/schema/exonum/consensus.proto (hunk 1 above)
Comment (Maria-Nosyk) [reply exonum/src/proto/schema/exonum/consensus.proto]:
```suggestion
  // Digital signature over `payload` created with `SecretKey` of the author of the message.
```

Comment (aleksuss) [reply exonum/src/proto/schema/exonum/consensus.proto]: I'm not sure if we really need a ` here. As I know the purpose of this docs is for a client code. @dmitry-timofeev what do you think ?

Comment (Maria-Nosyk) [reply exonum/src/proto/schema/exonum/consensus.proto]: alright, I do not mind to eliminate "`". In fact, I would even prefer that variant. In that case the comment should look like this:
> // Digital signature over the payload created with a secret key of the author of the message.

**Pull request merged**